pkg/azure/api/validation/validation_test.go

```diff
@@ -561,6 +561,26 @@ func TestValidateTags(t *testing.T) {
 	))
 }
 
+func TestCapacityReservationConfig(t *testing.T) {
```

Comment:
Could you please add one positive test case?

Reply:
Sure. I've added a positive case and pushed it.